src/MovieCard/MovieCard.js

```diff
@@ -16,20 +16,19 @@ class MovieCard extends React.Component{
   }
 
   render() {
-    console.log(this.state.popOut)
     return (
       <>
         {this.state.popOut
-            ? <div className='overlay' onClick={ () => this.setState( {popOut: null} ) }>
+            ? <div className='overlay' role='overlay' onClick={ () => this.setState( {popOut: null} ) }>
```

Comment:
I super dislike this kind of ternary. If you definitely want to use it, create a helper function called `displayOverlay` that returns the element, and place that as one value of the ternary, and create a second helper function called `displayMovieModal` that returns the MovieModal element, and make that function invocation the other value.

Reply:
Otherwise, just make two separate conditional render statements:
1 for if the popout is true
1 for if the popout is false